app/src/main/java/org/gdg/frisbee/android/chapter/InfoFragment.java

```diff
@@ -143,21 +152,15 @@ private void updateUIOnlineFrom(Person person) {
         }
     }
 
-    private void addOrganizers(final Person cachedChapter, boolean offline) {
```

Comment:
This is weird. I am sure that there were problems before. But now if we change this, we should go over all the usages of `addOrganizers`